lib/package-manager/install.js

```diff
@@ -31,6 +31,11 @@ function install(packageManager, context) {
       `Using temp directory: "${options.env['npm_config_tmp']}"`
     );
 
+    if (packageManager === 'npm') {
+      args.push('--no-audit');
```

Comment:
these could also be added in an .npmrc file

Reply:
We would have to automate creating a .npmrc and install it at the root of every module when we extract the tarball... seems like way more work that doing this.

Reply:
They’re read up the filesystem, so putting it in the root of this repo seems like it should be enough - but fair :-)

Reply:
That would not work with the architecture of CITGM. Every module is un archived into it's own file in the temp dir and `npm install` is run in that directory being orchestrated by CITGM